src/bioregistry/data/bioregistry.json

```diff
@@ -87464,6 +87491,31 @@
     "pattern": "^\\d{6}$",
     "uri_format": "https://abrc.osu.edu/stocks/number/SALK_$1"
   },
+  "salmon": {
+    "bioportal": {
+      "contact": {
+        "email": "mecum@nceas.ucsb.edu",
+        "name": "Bryce Mecum"
+      },
+      "description": "An ontology representing knowledge about salmon, features of their habitats, salmon stakeholders, and related entities. This ontology is considered to be in the early stages of development, though it is based upon considerable previous work. Contributions of all kinds are welcome and encouraged, including alignments, updates to existing terms, and new terms.",
+      "homepage": "https://github.com/DataONEorg/sem-prov-ontologies/",
+      "name": "Salmon Ontology",
+      "prefix": "SALMON",
+      "version": "0.3.1"
+    },
+    "contact": {
+      "email": "brycemecum@gmail.com",
+      "github": "amoeba",
+      "name": "Bryce Mecum",
+      "orcid": "0000-0002-0381-3766"
+    },
+    "example": "00000674",
+    "mappings": {
+      "bioportal": "SALMON"
+    },
+    "pattern": "^\\d+$",
+    "uri_format": "https://purl.dataone.org/odo/SALMON_$1"
```

Comment:
@amoeba is there a OWL file anywhere I can link to for SALMON or SASAP?

Reply:
I've merged this but am still keen to get this update in. Thanks for any help @mbjones